gci/models.py

```diff
@@ -1 +1,114 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Student(models.Model):
+
+    identifier = models.IntegerField(primary_key=True)
+    display_name = models.CharField(max_length=100)
+    program_year = models.ManyToManyField('ProgramYear')
+
+    def __str__(self):
+        return self.display_name
+
+
+class Organization(models.Model):
+
+    identifier = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=100)
+    logo = models.ImageField(upload_to='', blank=True)
+    url = models.URLField(blank=True)
+    summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Student)
+
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+
+    task_status = (
+        (1, 'Draft'),
+        (2, 'Published'),
+    )
+
+    name = models.CharField(max_length=200)
+    description = models.TextField(max_length=500)
```

Comment:
I think it is max 1500 ; does the API docs say something different?

Reply:
@jayvdb, It says 500;